Sources/GRPCCore/Documentation.docc/Documentation.md

```diff
@@ -64,3 +64,83 @@ Resources for developers working on gRPC Swift:
 
 - <doc:Design>
 - <doc:Benchmarks>
+
+### Client and Server
+
+- ``GRPCClient``
+- ``GRPCServer``
+- ``withGRPCClient(transport:interceptors:isolation:handleClient:)``
+- ``withGRPCClient(transport:interceptorPipeline:isolation:handleClient:)``
+- ``withGRPCServer(transport:services:interceptors:isolation:handleServer:)``
+- ``withGRPCServer(transport:services:interceptorPipeline:isolation:handleServer:)``
+
+### Request and response types
+
+- ``ClientRequest``
+- ``StreamingClientRequest``
+- ``ClientResponse``
+- ``StreamingClientResponse``
+- ``ServerRequest``
+- ``StreamingServerRequest``
+- ``ServerResponse``
+- ``StreamingServerResponse``
+
+### Service definition and routing
+
+- ``RegistrableRPCService``
+- ``RPCRouter``
+
+### Interceptors
+
+- ``ClientInterceptor``
+- ``ServerInterceptor``
+- ``ClientContext``
+- ``ServerContext``
+- ``ClientInterceptorPipelineOperation``
+- ``ServerInterceptorPipelineOperation``
```

Comment:
Just a reminder that this will have to change after the creation of `ConditionalInterceptor`

Reply:
The docs CI will catch it